### ISSUE TITLE
How to guide ideas

### ISSUE THREAD

**Issue by mattbrailsford** [vendrhub/vendr-documentation#41]
Just thought I'd jot down some how-to-guide ideas that could be nice to write and could be something people could help with should they like to.

- [ ] Single page checkout setup
- [ ] Sending custom emails
- [ ] Configuring a discount
- [ ] Capturing customizable order items (unique product properties)

Feel free to make suggestions in the comments and I'll update this list with suggestions. Also, do let me know if you'd like to tackle any of them.

**Comment by bjarnef** [comment]
Some other inspiration, which might be in separate tutorials and could include third party tools.

- Wishlist(s)
- Abandoned carts
- Product reviews (rating, name, comment). Content app on product node to see reviews in backoffice?

**Comment by mattbrailsford** [comment]
Ooh, awesome suggestions @bjarnef Ive added them to the lists. 

You could probably implement an abandoned cart system using Vendr’s email templates system and a background task. Should be ok for smaller solutions.

**Comment by bjarnef** [comment]
@mattbrailsford yeah, or probably using a third party tool like MailChimp, Campaign Monitor or Sleeknote and maybe in combination with Vendr email templates (or using email template from the external system). It would however be great with a example of this 👍

**Comment by bjarnef** [comment]
Adding a custom fee (or discount) would also be a great example.
A PR is available here https://github.com/vendrhub/vendr-documentation/pull/65 to add a custom deposit fee.
